src/AcceptanceTestHelper/AcceptanceTestHelper.csproj

```
@@ -12,4 +12,8 @@
     <ProjectReference Include="..\TestHelper\TestHelper.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="NServiceBus" Version="7.4.1" />
```

What would this look like for the merge to master with regards to the v8.0 release? Should we do the same as below? e.g. 

```suggestion
    <PackageReference Include="NserviceBus" Version="[7.4.0,)" />
```

---

@HEskandari Dependabot will update this repo anyway when a new NSB Core is released. Should we just be consistent with all of the testing tools?